strongbox-distribution/src/main/resources/setenv.conf

```diff
@@ -7,6 +7,8 @@ set.default.STRONGBOX_ORIENTDB_PROFILE=orientdb_EMBEDDED
 set.default.STRONGBOX_ORIENTDB_STUDIO_ENABLED=false
 set.default.STRONGBOX_ORIENTDB_STUDIO_IP_ADDRESS=127.0.0.1
 set.default.STRONGBOX_ORIENTDB_STUDIO_PORT=2480
-set.default.STRONGBOX_LOG_FILE=logback-spring.xml
+set.default.STRONGBOX_LOG_CONSOLE=false
+set.default.STRONGBOX_LOG_FILE=true
+set.default.STRONGBOX_DEBUG=false
```

Comment:
Just to clarify, setting `export STRONGBOX_DEBUG=true` would trigger debug logging, right?

Reply:
yes, there will be no `logback-spring-debug.xml` any more

Reply:
This is awesome! Will make things a lot more easier to manage in the docker container! Thanks for this improvement!

Reply:
yeah, I will provide all improvements within PR description

Reply:
Great! Ping me when it's ready and I'll add them to the docs.